components/BlurhashCanvas.vue

```diff
@@ -4,7 +4,8 @@
 
 <script lang="ts">
 import Vue from 'vue';
-import { decode } from 'blurhash';
+import Worker from 'worker-loader!./blurhash.worker';
+const worker = new Worker();
```

Comment:
This will create a new worker for each canvas (Since each canvas is a new component with its own lifecycle).

A better way to do this would be to have a generic web worker shared by the entire app (Created through a plugin, for example).

Generally, web workers should be reused as much as possible since, while not the most expensive thing around, they're still not instantaneous.

Reply:
I checked with a simple ``console.log()`` in the worker and this doesn't happen.

The portion of the code that gets executed everytime a new blurhash canvas is invoked starts in ``export default Vue.extend(`` line. 

If it worked the way you mention there would be a lot of overhead not only because of the workers, but also because of the constant imports ^^